css/media-queries.md

```diff
@@ -184,6 +184,8 @@ Since we have everything 1080px width and less specified, we’ll add something
 
 ### Check for Understanding
 
-1. Pair up with a peer and discuss what changes you would like to make to your recipe page project using media queries.
```

Comment:
Instead of deleting this, please move it a few lines above and title the section ### Challenge .

Reply:
Good catch!